docs/usage/configuration-options.md

```diff
@@ -10,15 +10,19 @@ Any config you define applies to the whole repository (e.g. if you have a monore
 
 You can store your Renovate configuration file in one of the following locations:
 
-- `.github/renovate.json`
-- `.github/renovate.json5`
-- `.gitlab/renovate.json`
-- `.gitlab/renovate.json5`
-- `.renovaterc.json`
-- `renovate.json`
-- `renovate.json5`
-- `.renovaterc`
-- `package.json` _(within a `"renovate"` section)_
+1. `renovate.json`
+1. `renovate.json5`
+1. `.github/renovate.json`
+1. `.github/renovate.json5`
+1. `.gitlab/renovate.json`
+1. `.gitlab/renovate.json5`
+1. `.renovaterc`
+1. `.renovaterc.json`
+1. `package.json` _(within a `"renovate"` section)_
```

Comment:
Can one of the maintainers check the order of the files? Some files have moved around in the list.

Reply:
https://github.com/renovatebot/renovate/blob/0b5ee0b40083644445cd5b42696d3f7eff94760a/lib/config/app-strings.ts#L1-L11

Thats the order

Reply:
Good! Thanks for checking! 😉

Reply:
This comment chain needs to be marked "resolved" by one of you with `write` rights before this will automerge properly. 😉